test/service_offering_test/service_offering_test.go

```diff
@@ -36,7 +36,7 @@ func TestServiceOfferings(t *testing.T) {
 
 var _ = test.DescribeTestsFor(test.TestCase{
 	API:            "/v1/service_offerings",
-	SupportsLabels: false,
+	SupportsLabels: true,
```

Comment:
As everything now supports labels, this field and the checks can be removed completely